modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/actions.es.js

```diff
@@ -96,31 +96,27 @@ export function populateConfigData([
 		};
 	});
 
+	const dataObject = dataObjects.find(({id}) => id === app.dataDefinitionId);
```

Comment:
Hey @naraandrade, Can you move this statement directly to where it is used?
`checkRequiredFields(formViews, dataObjects.find(({id}) => id === app.dataDefinitionId))`

Thx

Reply:
Never mind @naraandrade. I didn't see the other usage in the config.

Sorry :)

Reply:
So, using the _as used_ priciple we move it then.

Thx!